src/functions.rs

```diff
@@ -101,20 +101,39 @@ impl Functions {
     /// reference][`crate::reference::functions`].
     pub fn stdlib(ctx: &mut Context) -> Functions {
         let mut functions = Functions::new();
-        functions.add(ctx.add_identifier("child-index"), stdlib::ChildIndex);
-        functions.add(ctx.add_identifier("node"), stdlib::Node);
-        functions.add(ctx.add_identifier("plus"), stdlib::Plus);
-        functions.add(ctx.add_identifier("replace"), stdlib::Replace);
-        functions.add(ctx.add_identifier("source-text"), stdlib::SourceText);
-        functions.add(ctx.add_identifier("start-row"), stdlib::StartRow);
-        functions.add(ctx.add_identifier("start-column"), stdlib::StartColumn);
-        functions.add(ctx.add_identifier("end-row"), stdlib::EndRow);
-        functions.add(ctx.add_identifier("end-column"), stdlib::EndColumn);
-        functions.add(ctx.add_identifier("node-type"), stdlib::NodeType);
+        // general functions
+        functions.add(ctx.add_identifier("is-null"), stdlib::IsNull);
+        // tree functions
+        functions.add(
+            ctx.add_identifier("child-index"),
```

Comment:
One thing I've been wondering (and now seems like a good time to bring it up) is whether we should rename this to `named-child-index`.

I ended up requiring a function that got the index of _any_ child (including unnamed children), and had to call it `unnamed-child-index` because the more sensible name was taken already. Worse, `unnamed-child-index` is not even the right name (but I couldn't think of another, better name) because it also includes named children (by virtue of simply dispatching to the `tree-sitter` `children` method).

Reply:
I think that is a good idea. It describes its meaning better, and is also more consistent with `named-child-count`.